remix-analyzer/src/solidity-analyzer/modules/checksEffectsInteraction.ts

```diff
@@ -36,14 +40,14 @@ export default class checksEffectsInteraction implements AnalyzerModule {
                                 func)
                               )
       })
-      contract.functions.forEach((func) => {
+      contract.functions.forEach((func: FunctionHLAst) => {
         if (this.isPotentialVulnerableFunction(func, this.getContext(callGraph, contract, func))) {
-          const funcName = getFullQuallyfiedFuncDefinitionIdent(contract.node, func.node, func.parameters)
-          let comments = (hasModifiers) ? 'Note: Modifiers are currently not considered by this static analysis.' : ''
+          const funcName: string = getFullQuallyfiedFuncDefinitionIdent(contract.node, func.node, func.parameters)
```

Comment:
Same as abov about typing the return of `getFullQuallyfiedFuncDefinitionIdent` to string instead of hardtyping it here